modules/core/src/webgl/classes/renderbuffer.js

```diff
@@ -29,13 +30,22 @@ export default class Renderbuffer extends Resource {
 
   constructor(gl, opts = {}) {
     super(gl, opts);
+    this.gpuMemoryStats = statsManager.get('Memory Usage').get('GPU Memory');
```

Comment:
Would it make sense to define constants for these strings and decouple them from the printable representation? Or would that create too much coupling in the code?

Reply:
Or this could be done in the base `Resource` class constructor. We may also add a method `resource.setMemoryUsage` or something.

Reply:
Yeah, it would make sense to move at least parts of this into `Resource`. With the `byteLength` naming convention, it could easily manage everything except the per-type stats.